### PR TITLE
fix documentation

### PR DESCRIPTION
**Pull request by rle-mino**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,25 +19,15 @@ See [changelog](./CHANGELOG.md)
 ![PieChart exemple](/screenshots/socialchart.png)
 
 ```js
-import React, { Component } from 'react'
-import { PieChart } from 'components/piechart'
+import React, { Component } from 'react';
```
Comment (cedricdelpoux) [inline README.md]: Remove semicolon please

File: README.md [hunk 2]
```diff
@@ -19,25 +19,15 @@ See [changelog](./CHANGELOG.md)
...
+import PieChart from 'react-svg-piechart';
```
Comment (cedricdelpoux) [inline README.md]: Remove semicolon please

**Review by cedricdelpoux — CHANGES_REQUESTED** [review]

**Comment by cedricdelpoux** [comment]
Thank you for this PR. Remove the two semicolons please and I will merge it.

**Comment by rle-mino** [comment]
Done ! Thank you

File: README.md [hunk 3]
```diff
@@ -20,24 +20,14 @@ See [changelog](./CHANGELOG.md)
...
-  handleMouseEnterOnSector(sector) {
-    this.setState({ expandedSector: sector })
+  state = {
+    expandedSector: null,
   }
 
-  handleMouseLeaveFromSector() {
-    this.setState({ expandedSector: null })
-  }
+  handleMouseEnterOnSector = (sector) => this.setState({ expandedSector: sector });
```
Comment (cedricdelpoux) [inline README.md]: I'm sorry but you just add semicolons here

File: README.md [hunk 4]
```diff
@@ -46,33 +36,33 @@ export default class MyComponent extends Component {
       { label: 'Google Plus', value: 30, color: '#dd4b39' },
       { label: 'Pinterest', value: 20, color: '#cb2027' },
       { label: 'Linked In', value: 10, color: '#007bb6' },
-    ]
+    ];
+
+    const { expandedSector } = this.state;
```
Comment (cedricdelpoux) [inline README.md]: and here

**Review by cedricdelpoux — CHANGES_REQUESTED** [review]

**Comment by cedricdelpoux** [comment]
One semicolons removed, could you squash yours commits into one ? Thank you

**Comment by rle-mino** [comment]
Sorry, done !

**Comment by rle-mino** [comment]
Ok definitely, I'm sorry, this time should be the good one

**Comment by cedricdelpoux** [comment]
Thank you for your help

**Pull request merged**